api/src/json-schema/validation-schema.ts

```diff
@@ -165,6 +182,40 @@ export const submissionValidationSchema = {
       },
       additionalProperties: false
     },
+    workbook_parent_child_key_match_validator: {
+      description:
+        'Validates that this workbook submission file does not contain keys belonging to a child sheet that are missing in its parent sheet',
+      type: 'object',
+      properties: {
+        workbook_parent_child_key_match_validator: {
+          type: 'object',
+          required: ['child_worksheet_name', 'parent_worksheet_name', 'column_names'],
+          properties: {
+            name: {
+              type: 'string'
+            },
```

Comment:
nit: remove. This part of the original schema isn't the best, but the `name` was meant to be the name of the column the rule applies to, which doesn't matter in this case since its captured in the column_names field below

Reply:
Oh, I interpreted it as "the name of the parent-child-key relationship"; will remove